test_scripts/API/PerformAudioPassThru/025_ATF_P_PerformAudioPassThru_UI_SUCCESS_TTS_success_false_true.lua

```diff
@@ -0,0 +1,267 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PerformAudioPassThru] Speak (<errorCode>) + UI.PerformAudioPassThru (SUCCESS)
+-- SDL must transfer all <resultCodes> received from HMI to mobile app
+-- [HMI API] UI.PerformAudioPassThru request/response
+-- [HMI API] TTS.Speak request/response
+-- [Mobile API] PerformAudioPassThru request/response
+-- [HMI_API] [MOBILE_API] The "audioPassThruIcon" param at "ImageFieldName" struct
+--
+-- Description:
+-- In case SDL transfers Speak_request + UI.PerfromAudioPassThru_request to HMI
+-- and receives Speak(<errorCode>) or Speak (resultCode, success:true) response from HMI
+-- and UI.PerformAudioPassThru (SUCCESS) from HMI
+-- SDL must respond PerformAudioPassThru (WARNINGS, success:true) to mobile app
+--
+-- 1. Used preconditions
+-- 1.1. Structure containing all HMI result_codes, success: false + success: true (without SUCCESS itself) is created
+-- 1.2. PerformAudioPassThru RPC is allowed by policy
+-- 1.3. All params used in PerformAudioPassThru_request are present and within bounds
+-- 1.4. AudioPassThruIcon exists at apps sub-directory of AppStorageFolder (value from ini file)
+--
+-- 2. Performed steps
+-- Send PerformAudioPassThru (audioPassThruIcon, other params)
+-- HMI sends to SDL UI.PerformAudioPassThru (SUCCESS)
+-- HMI sends to SDL TTS.Speak (resultcode: HMI_result_code) 
+--
+-- Expected result:
+-- SDL sends UI.PerformAudioPassThru (audioPassThruIcon, other params) to HMI
+-- SDL sends  to HMI UI.PerformAudioPassThru
+-- SDL sends to mobile  TTS.Speak(resultcode: HMI_result_code, success:false/success:true)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPostconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPerformAudioPassThru = require('user_modules/shared_testcases/testCasesForPerformAudioPassThru')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+
+--[[ Local variables ]]
+-- info parameter is not specified in scope of the CRQ, it is intended for any applicable future use.
+local hmi_result_code_false = {
```

Comment:
@maria-mih, you can merge all result codes in one structure, because expected result is the same for success true and success false.
For this structure can use additional element: success = false/true